deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eu
+
+rm -rf .terraform*
```

Comment:
wouldn't the opta cli do this anyway?

Reply:
yea, i kinda of just copy-pasted, but you're right. I'll remove this line